src/main/java/org/polyfill/components/Polyfill.java

```diff
@@ -135,6 +137,35 @@ public int hashCode() {
         return hash;
     }
 
+    /**
+     * Gets the type of license of the polyfill implementation e.g. MIT | Apache
+     * @return {String} - The type of license
+     */
+    public String getLicense() {
+        return (String) this.polyfillMap.get(LICENSE);
+    }
+
+    /**
+     * Gets the URL to the polyfill repository
+     * @return {String} - The URl to the repository
+     */
+    public String getRepository() {
+        return (String) this.polyfillMap.get(REPO);
+    }
+
+    public Map<String, Object> toMap() {
```

Comment:
I think we should have this somewhere else. Polyfill should stay simple and should not contain knowledge about what to return in toMap. The most we can do is to return this.polyfills.clone() and let remove the unnecessary fields somewhere else.